Resources/views/ContentType/update_content_type.html.twig

```diff
@@ -0,0 +1,97 @@
+{% extends "eZPlatformUIBundle::pjax_admin.html.twig" %}
+{# @var contentTypeDraft \eZ\Publish\API\Repository\Values\ContentType\ContentTypeDraft #}
+{# @var fieldTypeMapper \EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperRegistryInterface #}
```

Comment:
It's to get code completion with phpstorm: http://symfony2-plugin.espend.de/languages/twig/index.html#phptypes